src/routes/(infoPages)/[topTierPage]/[...serviceGroupPage]/+page.svelte

```diff
@@ -247,3 +251,21 @@
     imageSizeType="col-9"
   />
 {/if}
+
+{#if news?.items && news.items.length > 0}
+  <h2>Recent news</h2>
+  {#each news.items as entry (entry?.sys.id)}
+    {#if entry}
+      <NewsEntrySnippet {entry} headingLevel={4} />
+    {/if}
+  {/each}
+{/if}
+
+{#if events && events.items.length > 0}
```

Comment:
I think we want this instead of consistency with `news` above, yeah?
```suggestion
{#if events?.items && events.items.length > 0}
```